diff --git a/private/usage/usage.go b/private/usage/usage.go
index 9cac181..1b00d09 100644
--- a/private/usage/usage.go
+++ b/private/usage/usage.go
@@ -14,31 +14,6 @@
 
 package usage
 
-import (
-	"errors"
-	"fmt"
-	"os"
-	"path/filepath"
-	"runtime/debug"
-	"strings"
-)
-
 func init() {
-	if err := check(); err != nil {
-		panic(err.Error())
-	}
-}
-
-func check() error {
-	buildInfo, ok := debug.ReadBuildInfo()
-	if !ok {
-		if !strings.HasSuffix(os.Args[0], testSuffix) && filepath.Base(os.Args[0]) != debugBin {
-			return errors.New("github.com/bufbuild/buf/private code must only be imported by github.com/bufbuild projects")
-		}
-		return nil
-	}
-	if !strings.HasPrefix(buildInfo.Main.Path, "github.com/bufbuild") {
-		return fmt.Errorf("github.com/bufbuild/buf/private code must only be imported by github.com/bufbuild projects but was used in %s", buildInfo.Main.Path)
-	}
-	return nil
+	// nothing to see here
 }
